tests/integration/classification_test.py

```diff
@@ -4,41 +4,50 @@
 
 from pyatlan.cache.classification_cache import ClassificationCache
 from pyatlan.client.atlan import AtlanClient
+from pyatlan.model.enums import AtlanClassificationColor, AtlanTypeCategory
 from pyatlan.model.typedef import ClassificationDef
 
-CLS_NAME = "psdk-ClassificationTest"
+CLS_NAME = "psdk-classification"
 
-# NOTE: Tests are intended to run in the order specified in this file,
-# to complete creation prior to testing prior to deletion.
 
+class ClassificationTest:
+    @staticmethod
+    def create_classification(client: AtlanClient, name: str) -> ClassificationDef:
+        classification_def = ClassificationDef.create(
+            name=name, color=AtlanClassificationColor.GREEN
+        )
+        response = client.create_typedef(classification_def)
+        assert response
+        assert len(response.classification_defs) == 1
+        created = response.classification_defs[0]
+        assert created.category == AtlanTypeCategory.CLASSIFICATION
+        assert created.name
+        assert created.guid
+        assert created.name != name
+        assert created.display_name == name
+        return created
 
-@pytest.fixture
+    @staticmethod
+    def delete_classification(client: AtlanClient, cls: ClassificationDef) -> None:
+        client.purge_typedef(cls.name)
+
+
+@pytest.fixture(scope="session")
 def client() -> AtlanClient:
     return AtlanClient()
 
 
-def test_001_create_classification(client: AtlanClient):
-    cls = ClassificationDef(
-        name=CLS_NAME, display_name=CLS_NAME, options={"color": "GREEN"}
-    )
-    response = client.create_typedef(cls)
-    print(response)
-    assert response
-    assert response.classification_defs
-    assert len(response.classification_defs) == 1
+@pytest.fixture(scope="module")
+def cls(client: AtlanClient):
```

Comment:
Since this fixture returns a classification the convention would be to name is classification

Reply:
Actually returns a `classification_def`, so will name it that instead (and also add the missing return type hint 🙈)